### DISCUSSION TITLE
Discussion categories

### DISCUSSION THREAD

**Discussion by maratori** [lona-web-org/lona#100] [General]
There is a custom discussion category `Feature Request` and it has format `Question and Answer` now.
I think it's better to change format to `Open-ended discussion`.
Link to [github documentation](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository).

Q&A is something like stackoverflow alternative.

**Comment by fscherf** [comment]
@maratori Ack! Thanks for bringing this up!
